src/main/java/ConsolePrinter.java

```diff
@@ -0,0 +1,8 @@
+public class ConsolePrinter {
+    public Integer printValue() {
+        Integer intValue = 1000;
```

Comment:
Move `Integer intValue` to object properties. It should be described outside of the method `printValue()`

Reply:
Done